frameworks/g-cloud-9/questions/services/supportedBrowsers.yml

```diff
@@ -0,0 +1,37 @@
+name: Supported browsers
+question: 'What browsers does your service work with?'
+depends:
+  -
+    "on": lot
+    being:
+      - cloud-hosting
+      - cloud-software
+      - cloud-support
+hint: 'Choose all that apply.'
+type: checkboxes
+options:
+    -
+        label: 'Internet Explorer 6'
+    -
+        label: 'Internet Explorer 7'
+    -
+        label: 'Internet Explorer 8'
+    -
+        label: 'Internet Explorer 9'
+    -
+        label: 'Internet Explorer 10+'
+    -
+        label: 'Microsoft Edge'
+    -
+        label: Firefox
+    -
+        label: Chrome
+    -
+        label: Safari
+    -
+        label: Opera
```

Comment:
We’ve added questions 'Has your service been designed to work on mobile devices?', if Yes 'Describe any differences between the mobile and desktop service'. So we’re not calling out mobile browsers but we are asking about mobile. The browser list is the same as Service Manual https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices